go/libraries/doltcore/sqle/logictest/DoltReleaseBenchmarkJenkinsfile

```diff
@@ -0,0 +1,89 @@
+pipeline {
+    agent none
+    stages {
+        stage('Test Dolt release') {
+            parallel {
+                stage ("Update Liquidata/dolt-sql-performance:releases") {
+                    agent {
+                        kubernetes {
+                            label "liquidata-inc-ld-build"
+                        }
+                    }
+                    environment {
+                        PATH = "${pwd()}/.ci_bin:${pwd()}/.ci_bin/node_modules/.bin:${env.PATH}"
+                        TMPDIR = "${pwd()}/tempDir"
+                        CREDSDIR = "/home/jenkins/.dolt/creds"
+                        DOLT_CREDS = credentials("system-account-dolthub-creds")
+                        DOLT_GLOBAL_CONFIG = credentials("system-account-dolthub-config")
+                        DOLT_CONFIG_PATH = "/home/jenkins/.dolt"
+                    }
+                    steps {
+                        dir ("sqllogictest") {
+                            git url: "https://github.com/liquidata-inc/sqllogictest.git"
+                        }
+                        dir ("dolt-release") {
+                            checkout([$class: 'GitSCM', branches: [[name: "${DOLT_RELEASE}"]],
+                                 userRemoteConfigs: [[url: 'https://github.com/liquidata-inc/dolt.git']]])
+                        }
+                        dir ("dolt-release/go") {
```

Comment:
Instead of building from source, should we download it from the github releases page? Where does this fit in the release workflow?